pkgs/by-name/li/libyuv/package.nix

```diff
@@ -17,6 +17,11 @@ stdenv.mkDerivation {
     hash = "sha256-4Irs+hlAvr6v5UKXmKHhg4IK3cTWdsFWxt1QTS0rizU=";
   };
 
+  patches = [
```

Comment:
We can add the patch only when `stdenv.hostPlatform.isBigEndian`, no need to mass rebuild.